lib/src/library_scanner.dart

```diff
@@ -48,9 +48,11 @@ class LibraryScanner {
           'expectetd location.\n$dotPackagesPath');
     }
 
+    // TODO: figure out why non-flutter pub list doesn't work the same way as the default
```

Comment:
The pub client that's part of the flutter SDK "understands" flutter – it does the right thing when it sees a flutter dependency, for example